app/panel/components/Subscription.jsx

```diff
@@ -46,6 +50,27 @@ class Subscription extends React.Component {
 			this.props.history.push('/detail');
 		}
 	}
+	parseSubscriptionData = () => {
+		const sd = this.props.subscriptionData;
+		if (sd) {
+			const languageToken = this.props.language.toLowerCase().replace('_', '-');
+			moment.locale(languageToken);
+			const {
+				planAmount, planCurrency, planInterval, currentPeriodEnd, cancelAtPeriodEnd, status
+			} = sd;
+			const plan_amount = Math.floor(planAmount / 100);
+			const plan_ends = cancelAtPeriodEnd ? moment.duration(moment.unix(currentPeriodEnd).diff(moment(new Date()))).days() : '';
+			return {
+				plan_amount: new Intl.NumberFormat(languageToken, { style: 'currency', currency: planCurrency }).format(plan_amount),
+				plan_interval: planInterval,
+				active: (status === 'active'),
+				charge_date: moment.unix(currentPeriodEnd).format('MMMM Do, YYYY'),
+				plan_ends,
+				loading: false,
+			};
+		}
+		return { loading: true };
+	}
```

Comment:
@zarembsky we don't need to localize the currency since we're only taking USD for the time being, please remove it.

Reply:
Implemented